example-app/README.md

```diff
@@ -0,0 +1,25 @@
+# Example app
+
```

Comment:
What do you think of adding a small description of what this module is? Just to clarify that it is for testing purposes and should not be used as a template for integrating with Adyen? Maybe we can link to the docs to explain that this should be their main reference?